### PR TITLE
Add forgetLocalItem and forgetItem test

### PR DESCRIPTION
**Pull request by Hjortsberg**
Should also be coupled into the DHT

### CODE REVIEW

File: store/store_test.go [hunk 1]
```diff
@@ -285,3 +285,25 @@ func TestLocalItemCh(t *testing.T) {
...
+
+	trueHash := [32]byte{174, 79, 167, 92, 82, 249, 190, 142, 129, 67, 178, 149, 52, 212, 158, 150, 67, 136, 83, 10, 170, 233, 83, 34, 158, 194, 62, 241, 14, 168, 19, 103}
+
+	testVal := "q"
+
+	db.AddLocalItem(trueHash, testVal)
+
+	_, err := db.GetLocalItem(trueHash)
+	if err != nil {
+		t.Errorf("expected localItem to be in db")
```
Comment (optmzr) [inline store/store_test.go]:
```suggestion
		t.Error("expected localItem to be in db")
```

File: store/store_test.go [hunk 2]
```diff
@@ -285,3 +285,25 @@ func TestLocalItemCh(t *testing.T) {
...
+	}
+
+	db.ForgetItem(trueHash)
+
+	_, err = db.GetLocalItem(trueHash)
+	if err == nil {
+		t.Errorf("expected localItem is still in db, expected error")
```
Comment (optmzr) [inline store/store_test.go]:
```suggestion
		t.Error("expected localItem is still in db, expected error")
```

File: ctl/ctl.go [hunk 3]
```diff
@@ -56,6 +60,10 @@ func (a *API) Get(get Get, reply *GetReply) (err error) {
 	return
 }
 
+func (a *API) Forget(forget Forget) {
+	log.Info().Msgf("Forget: %s", forget.Key)
+}
```
[the remark this answers is not in the document]
Comment (Hjortsberg) [reply ctl/ctl.go]: honestly, I don't know 🙄. Is this part for the REST API?

[the remark this answers is not in the document]
Comment (optmzr) [reply ctl/ctl.go]: No, this is the RPC API, this is what is called when you do a `dhtnode -forget <key>`

[the remark this answers is not in the document]
Comment (Hjortsberg) [reply ctl/ctl.go]: Yeah ok, that command seems to work. Do you mean i should implement that part in another PR ?

[the remark this answers is not in the document]
Comment (optmzr) [reply ctl/ctl.go]: Per IRL discussion this should be done in this PR.

**Pull request merged**